class A extends Exception {}

class athrow1 {
    //@ ensures \result <==> false;
  public static boolean func() {
    A a = new A();
    try {
      throw a;
    } catch (Exception e) {
      if(true) return false;
    }
    return true;
  }
}

